README.md

```diff
@@ -503,23 +506,25 @@ Author of this script is [amargedon](https://github.com/amargedon).
 
 This is the layout of the config file used by the `trailingstoploss_tp.py` bot helper:
 
--   **timezone** - timezone. (default is 'Europe/Amsterdam')
--   **check-interval** - update interval in Seconds when no deals with SL are active. (default is 120)
--   **monitorinterval** - update interval in Seconds when there are deals with SL active. (default is 60)
--   **debug** - set to true to enable debug logging to file. (default is False)
--   **logrotate** - number of days to keep logs. (default = 7)
--   **3c-apikey** - your 3Commas API key value.
--   **3c-apisecret** - your 3Commas API key secret value.
--   **notifications** - set to true to enable notifications. (default = False)
--   **notify-urls** - one or a list of apprise notify urls, each in " " seperated with commas. See [Apprise website](https://github.com/caronc/apprise) for more information.
--   *[tsl_tp_]*
--   **botids** - a list of bot id's to manage separated with commas.
--   **config** - a list of objects with the settings for that percentage.
--   *object*
--   **activation-percentage** - from % of profit this object is valid for.
--   **initial-stoploss-percentage** - % of stoploss to set when activation-percentage is reached.
--   **sl-increment-factor** - % to increase the SL with, based on % profit after activation-percentage.
--   **tp-increment-factor** - % to increase the TP with, based on % profit after activation-percentage.
+- **timezone** - timezone. (default is 'Europe/Amsterdam')
+- **check-interval** - update interval in Seconds when no deals with SL are active. (default is 120)
+- **monitorinterval** - update interval in Seconds when there are deals with SL active. (default is 60)
+- **debug** - set to true to enable debug logging to file. (default is False)
+- **logrotate** - number of days to keep logs. (default = 7)
+- **3c-apikey** - your 3Commas API key value.
+- **3c-apisecret** - your 3Commas API key secret value.
+- **notifications** - set to true to enable notifications. (default = False)
+- **notify-urls** - one or a list of apprise notify urls, each in " " seperated with commas. See [Apprise website](https://github.com/caronc/apprise) for more information.
+- *[tsl_tp_]*
+- **botids** - a list of bot id's to manage separated with commas.
+- **config** - a list of objects with the settings for that percentage.
+- *object*
+- **activation-percentage** - from % of profit this object is valid for.
+- **initial-stoploss-percentage** - % of stoploss to set when activation-percentage is reached.
+- **sl-timeout** - timeout before stoploss is triggered. 3Commas still handles triggering of the stop loss, but the market buy order is only triggerd after the timeout has expired and the current price is still under the stop loss price. ***(Be careful, the market order will also be activated when the profit is below 0%. This means the deal could end up selling at a loss!)***  (Default is 0 which means it is turned off). 
```

Comment:
Dit staat boven in de tekst al uitgelegd, dus ik zou dit niet twee keer erin willen hebben. Dit lijstje is ook als kort overzicht bedoeld, dus zou eigenlijk iets als 'stoploss timeout in seconds. (default = 0)' verwachten.